src/pages/StudyManage/index.tsx

```diff
@@ -31,33 +61,82 @@ function StudyManage() {
     return title;
   };
 
+  const removeStudy = async (
+    studyId: string,
+    e: React.MouseEvent<HTMLAnchorElement>,
+  ) => {
+    try {
+      await deleteStudy(studyId);
+      dispatch(
+        openAlert({
+          severity: 'success',
+          title: '삭제되었습니다!',
+          content: '스터디를 성공적으로 삭제했습니다!',
+        }),
+      );
+      handleClickLink(e);
```

Comment:
`handleClickLink`는 `useLinkClickHandler('/')`로 스터디 삭제 후 메인으로 이동하시기 위해서 사용하신 것으로 이해했습니다!

단순히 스터디 삭제 후 `url` 이동 목적이라면 `useNavigate()` Hook을 이용해서 아래처럼 수정하면 될 것 같은데욥, `useLinkClickHandler`를 사용하신 목적이 따로 있으신가욥?! 궁금해서 여쭤봅니다!

```suggestion
      navigate('/', { replace: true });
```

Reply:
이부분은 link의 시맨틱 a태그를 유지하기 위해서 사용한건데요 처음에는 navigate로 할려다가 저런 훅이 존재하길래 사용해보았습니다~